### PR TITLE
Web form

### LINKED ISSUE #19: Add Login Page Web form
Make the login page be able to take in inputs of username and password.

### PR DESCRIPTION
**Pull request by jlc1541**
Uses Flask Extension of wtforms to represent the web form.
A base html file is used as the structure to the other template html forms such as the login.html as in they are extensions of the base.
A login view function is included to display the forms correctly.

### CODE REVIEW

File: app/routes.py [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
+from app import app
+from app.forms import LoginForm
+
+@app.route('/')
+@app.route('/test')
+def test():
+    user = {'username' : 'User'}
+    return render_template('test.html', title = 'Home', user = user)
+
+@app.route('/login', methods=['GET','POST'])
```
Comment (haleyhagler227) [inline app/routes.py]: Make this route ('/') that way it is the first page seen when viewing this

File: app/templates/test.html [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <h1>Hi, {{user.username}}!</h1>
+    {% for post in posts %}
+    <div><p>{{ post.author.username }} says: <b>{{ post.body }}</b></p></div>
+    {% endfor %}
```
Comment (haleyhagler227) [inline app/templates/test.html]: There are no post in the app. These lines should be deleted

**Review by haleyhagler227 — CHANGES_REQUESTED** [review]
Please delete the binary files. They end with cpython-37.pyc. I commented on a few other changes that should be made.

File: app/routes.py (hunk 1 above)
Comment (jlc1541) [reply app/routes.py]: Just made changes which can be viewed now.

**Review by haleyhagler227 — APPROVED** [review]

**Comment by FullmetalPotato** [comment]
Its better now that the conflicts have been resolved. A good starting point for logging in.

**Pull request merged**